services/slam/builtin/orbslam_yaml_test.go

```diff
@@ -116,7 +117,12 @@ func TestOrbslamYAMLNew(t *testing.T) {
 	t.Run("New orbslamv3 service with good camera and defined params", func(t *testing.T) {
 		// Create slam service
 		logger := golog.NewTestLogger(t)
-		grpcServer := setupTestGRPCServer(attrCfgGood.Port)
+		reservePort, err := utils.TryReserveRandomPort()
+		test.That(t, err, test.ShouldBeNil)
+		attrCfgGood.Port = "localhost:" + strconv.Itoa(reservePort)
+		grpcServer, err := setupTestGRPCServer(t, attrCfgGood.Port)
+		test.That(t, err, test.ShouldBeNil)
+		time.Sleep(time.Millisecond * 1)
```

Comment:
What is the reason for this sleep?

Reply:
no reason, missed it when I was cleaning up code